src/foam/nanos/analytics/mixpanel/MixpanelDeliveryThread.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.analytics.mixpanel',
+  name: 'MixpanelDeliveryThread',
+
+  documentation: `
```

Comment:
Is the goal to send mix panel messages as one bulk message instead of each individually?